src/helpers/search.js

```diff
@@ -76,8 +84,20 @@ export class SuggestionsHelper {
   getSuggestions() {
     return this.suggestions;
   }
+
 }
 
+const localeCompareWithYear = (a, b) => {
+  const aSliced = a.suggest.split(a.year).map(w => w.trim());
+  const bSliced = b.suggest.split(b.year).map(w => w.trim());
+
+  if (aSliced[0] && aSliced[0].toLowerCase() !== bSliced[0].toLowerCase())
```

Comment:
You should check bSliced[0] to exists, same as aSliced[0]

Reply:
It can exist but it can be empty string, this that I check for aSliced[0]
bu if 
bSliced[0] === "" 
this will be false
aSliced[0].toLowerCase() !== bSliced[0].toLowerCase()